src/main/resources/db/changelog/db.changelog-master.yml

```diff
@@ -59,6 +59,9 @@ databaseChangeLog:
   - include:
       file: changes01/09072021_insert_user_image_url.yml
       relativeToChangelogFile: true
+  - include:
```

Comment:
This change set should be added as the last one. Because the next one (init/13072021_add_latitude_and_longitude_columns_to_location_table.yml) is already executed. The order matters here

Reply:
changed orded